packages/edit-site/src/components/navigation-sidebar/navigation-toggle/index.js

```diff
@@ -5,30 +5,54 @@ import { useDispatch, useSelect } from '@wordpress/data';
 import { Button, Icon } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { wordpress } from '@wordpress/icons';
+import { store as coreDataStore } from '@wordpress/core-data';
 
 /**
  * Internal dependencies
  */
 import { store as editSiteStore } from '../../../store';
+import {
+	MENU_TEMPLATE_PARTS,
+	MENU_TEMPLATES,
+} from '../navigation-panel/constants';
 
 function NavigationToggle( { icon, isOpen } ) {
-	const { isRequestingSiteIcon, siteIconUrl } = useSelect( ( select ) => {
-		const { getEntityRecord } = select( 'core' );
-		const { isResolving } = select( 'core/data' );
-		const siteData =
-			getEntityRecord( 'root', '__unstableBase', undefined ) || {};
+	const { isRequestingSiteIcon, templateType, siteIconUrl } = useSelect(
+		( select ) => {
+			const { getEditedPostType } = select( editSiteStore );
```

Comment:
You need to update the [tests](https://github.com/WordPress/gutenberg/blob/cffac592c49776955a8eabebd16dcd1da946fba7/packages/edit-site/src/components/navigation-sidebar/navigation-toggle/test/index.js). No mock is provided for this select.